### ISSUE TITLE
Could not figure out how to render a `Model` as in the README.md example

### ISSUE THREAD

**Issue by eordano** [brianzinn/react-babylonjs#33]
Hi there, thanks for this library! It's really cool. I can see myself using it a lot in the future :D

Only caveat is that I could not get a model to render, as I couldn't figure out a way to type it. I also had some difficulty finding how are you dealing and loading models in the code.

I was able to force it to type doing:
```
declare global {
  namespace JSX {
    interface IntrinsicElements {
      'model': any
    }
  }
}
```

In particular, my problem is that `<model>` is not being catched by `react-reconciler`:
<img width="417" alt="Screen Shot 2019-12-29 at 09 27 38" src="https://user-images.githubusercontent.com/42750/71556939-798e2b80-2a1d-11ea-8339-f352153ebb6b.png">

Also, I tried it in uppercase (hacking through:
`
declare global {
  var Model: any
  namespace JSX {
    interface IntrinsicElements {
      'Model': any
    }
  }
}
`) but it wasn't enough (`Model` never gets imported and it fails in runtime).

Could you advice with a better example than the one found in the storybook? The use of `WithModel` and the hidden file outside of the story makes it a very hard example to follow. Let me know if I can help in any way.

**Comment by eordano** [comment]
Nevermind. Sorry, I was using it outside of a babylon context!

**Comment by brianzinn** [comment]
Thanks @eordano for reporting. I'll add `model` to the intrinsic elements and link through to the `Model` component for consistency.  My intention was to separate what is available through the Babylon API via generated code from the custom components that I have created.

I'll look at fixing up the storybook examples, so they have the full code instead of referencing components!  I did at one point start on that and then hit a snag, but will try again.  Thanks for pointing that out.  If you have any other feedback, it is much appreciated.  Cheers.
